stable/magic-namespace/values.yaml

```
@@ -23,6 +23,9 @@ tiller:
 
   maxHistory: 0
 
+  ## Override the default 'configmap' Tiller storage method
+  # storage: secret
```

In general I prefer setting default values explicitly, because this provides more information to users when running commands like `helm get [release]` and inspecting computed values.

```suggestion
  storage: configmap
```

---

On a side note, I would be strongly in favor of making `secret` the default. I imagine this would need to be a major version bump since upgrading in place may require additional migration steps (eg. [tiller-releases-converter](https://github.com/dragonsmith/tiller-releases-converter)).

---

Thanks for your suggestions @jlegrone, I implemented them all. Also happy to do this if you'd prefer. If so, I've got a bugfix regarding the secret name I believe should really go on first.

---

@acaire I'm good with merging this as is for now, and making that breaking change separately (with good migration instructions). Please feel free to add me as a reviewer on additional PRs!

---

Works for me, thanks!